README.md

```diff
@@ -5,3 +5,57 @@
 ```sbt
 libraryDependencies += "com.precog" %% "quasar-datasource-kafka" % <version>
 ```
+
+## Configuration
+
+The configuration of the Kafka Datasource has the following JSON format:
+
+```
+{
+ "bootstrapServers": Array of Strings "host:port",
+ "groupId": String,
+ "topics": Array of Strings,
+ "decoder": < "RawKey" | "RawValue" >,
+  "format": {
+    "type": "json" | "separated-values"
+    // for "json"
+    "precise": Boolean,
+    "variant" "array-wrapped" | "line-delimited"
+    // for "separated-values", all strings must be one symbol length
+    "header": Boolean,
+    // The first char of row break
+    "row1": String,
+    // The second char of row break, empty string if row break has only one symbol
+    "row2": String,
+    // Column separator (char)
+    "record": String,
+    "openQuote": String,
+    "closeQuote": String,
+    "escape": String
+  },
+}
```

Comment:
Confirming that this version of the connector does not support gzipped data?

Reply:
If the data stored on kafka is gzipped data, the connector will pass it on to quasar without any issues. The connector doesn't care at all what the format is, since the data is opaque to it.

Reply:
In order to pass the gzipped data on to quasar, it'll need to do so using `DataFormat.Compressed`. This is represented in the json config [using the field](https://github.com/precog/quasar/blob/v180.1.0/connector/src/main/scala/quasar/connector/DataFormat.scala#L124) `"compressionScheme"`. And so this field will need to be present in the datasource config. For examples, see `quasar-datasource-s3` and `quasar-datasource-url`.